src/server/game/AI/CreatureAIImpl.h

```diff
@@ -58,12 +58,7 @@ bool InstanceHasScript(WorldObject const* obj, char const* scriptName);
 template<class AI, class T>
 inline AI* GetInstanceAI(T* obj, char const* scriptName)
 {
-    if (InstanceHasScript(obj, scriptName))
-    {
-        return new AI(obj);
-    }
-
-    return nullptr;
+return InstanceHasScript(obj, scriptName) ? new AI(obj) : nullptr
```

Comment:
indentation

Reply:
add 4 whitespace and `;` in end

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/4596#discussion_r604199917 I blame shin D: :P